tools/kwiver_applet.h

```diff
@@ -33,6 +33,8 @@
 
 #include <tools/kwiver_tools_applet_export.h>
```

Comment:
Maybe this is for a later branch, but we should move all framework code into `vital` or `vital/applet`.  The `tools` directory should only contain cxx files that get built into executables.   We should never have `tools` on the include path.

Reply:
This can be resolved in a later PR.